add blog for PyPI Support Specialist (#15645)
pypi/warehouse: docs/blog/posts/2024-03-20-announcing-a-pypi-support-specialist.md

there are over a half a million packages [on PyPI](https://pypi.org/).
The impact PyPI has these days is pretty breathtaking.
Running a free public service of that size has come with challenges, too.
As PyPI has grown, the work of communicating with users
and solving account issues here has grown in tandem
and out-stripped our current volunteer plus
one tenth of a staff person capacity.
We also know that some community members have noticed
and expressed frustration with the time-frame that goes with tasks
that don't have sufficient staffing. 

Much of this work is sensitive and complex such that
it needs to be performed by a PSF staff person.
It involves personal information and verification processes
to make sure we’re giving access and names to the correct entities.
Work like this needs to be done by a person who is here day after day
to carry out multi-step verification procedures
and is accountable to the PSF. 

We are very happy to share the news that
[we are hiring a person](https://pythonsoftwarefoundation.applytojob.com/apply/nyYHuOha9h/PyPI-Support-Specialist)
to help us manage the increased capacity
and allow us to keep pace with PyPI’s seemingly unstoppable growth.

> This is a junior role that is 100% remote.

Reviewer: ```suggestion
This is an entry-level role that is 100% remote.
```
or
```suggestion
This is an associate role that is 100% remote.
```
